airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml

```diff
@@ -0,0 +1,1609 @@
+                        type: boolean
+                      profile_content:
+                        type: string
+                      project_type:
+                        type: string
+                      public:
+                        type: boolean
+                      show_priority_icon:
+                        type: boolean
+                      show_priority_icon_in_all_panels:
+                        type: boolean
+                      show_story_priority:
+                        type: boolean
+                      start_time:
+                        type: string
+                      time_zone:
+                        properties:
+                          kind:
+                            type: string
+                          offset:
+                            type: string
+                          olson_name:
+                            type: string
+                        type: object
+                      updated_at:
+                        type: string
+                      velocity_averaged_over:
+                        type: number
+                      version:
+                        type: number
+                      week_start_day:
+                        type: string
+                    type: object
+                retriever:
+                  type: SimpleRetriever
+                  requester:
+                    type: HttpRequester
+                    url_base: https://www.pivotaltracker.com/services/v5/
+                    path: projects
+                    http_method: GET
+                    request_parameters: {}
+                    request_headers: {}
+                    authenticator:
+                      type: ApiKeyAuthenticator
+                      api_token: "{{ config['api_token'] }}"
+                      inject_into:
+                        type: RequestOption
+                        inject_into: header
+                        field_name: X-TrackerToken
+                    request_body_json: {}
+                  record_selector:
+                    type: RecordSelector
+                    extractor:
+                      type: DpathExtractor
+                      field_path: []
+                  paginator:
+                    type: DefaultPaginator
+                    page_token_option:
+                      type: RequestOption
+                      inject_into: request_parameter
+                      field_name: offset
+                    pagination_strategy:
+                      type: CursorPagination
+                      cursor_value: >-
+                        {{ response['headers',
+                        'X-Tracker-Pagination-Offset']+response['headers',
+                        'X-Tracker-Pagination-Limit' ]}}
+                      stop_condition: >-
+                        {{ 'X-Tracker-Pagination-Total' not in
+                        response['headers'] }}
+  - type: DeclarativeStream
+    name: activity
```

Comment:
Missing transformation from the Python implementation - please add as described [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/source.py#L103)

Reply:
@btkcodedev field `project_id` is not present in records